test/helpers/maintenance_tasks/tasks_helper_test.rb

```diff
@@ -91,6 +91,18 @@ class TasksHelperTest < ActionView::TestCase
         csv_file_download_path(run)
     end
 
+    test "#resolve_inclusion_value resolves enum_attr" do
```

Comment:
This test already passes on main. We should probably use a separate `attribute` that uses a proc or symbol for `:in` and see it fail.

Reply:
I'm fine with keeping this test, but agree that we should add a new test for the case we're fixing 👍`enum_attr` in the test name is a bit vague. Perhaps we could say something like `test "#resolve_inclusion_value resolves inclusion validator for task attribute"` ?

Reply:
Yes, that makes sense -- related to your [point here](https://github.com/Shopify/maintenance_tasks/pull/962#discussion_r1482809233), that we only render the view for the first `validates_inclusion_of`. This test would only fail on main if we have the proc or symbol as the first validation:
```ruby
    # put the proc validates_inclusion_of before the array one
    validates_inclusion_of :enum_attr, in: proc { [100, 200, 300] }, allow_nil: true
    validates_inclusion_of :enum_attr, in: [100, 200, 300], allow_nil: true
```
(Fails on main)